beaconChain/src/helpers/stateTransitionHelpers.ts

```diff
@@ -19,27 +23,37 @@ import {
 import {
   AttestationData,
   AttestationDataAndCustodyBit,
+  BLSSignature,
   BeaconState,
   bool,
   bytes,
   bytes32,
+  DepositInput,
   Epoch,
   Fork,
+  Gwei,
   int,
   Shard,
   SlashableAttestation,
   Slot,
   uint64,
   Validator,
   ValidatorIndex,
+  BLSPubkey,
 } from "../types";
 
 import {
   blsAggregatePubkeys,
   blsVerifyMultiple,
-} from "../lib/stubs/bls";
+  blsVerify,
+} from "../stubs/bls";
 
-import { hashTreeRoot } from "../lib/state";
+
+// This function was copied from ssz-js
+// TODO: either export hash from ssz-js or move to a util-crypto library
```

Comment:
Let's export from SSZ-JS and cut a new release in a week

Reply:
The block processing PR already imports from ssz-js. I figured we may as well use it if we got it, rather than importing and using two different places, the stub and the real thing.

Reply:
Makes sense.